src/Event.scala

```diff
@@ -0,0 +1,10 @@
+/**
+  * Created by halleyfroeb on 10/17/16.
+  */
+case class Event (val name: String, val date: String, val location: String) {
```

Comment:
FYI, with a case class the default is "val" so you don't need to specify unless you want to use "var".